contracts/consensus/Consensus.sol

```diff
@@ -654,17 +669,18 @@ contract Consensus is MasterCopyNonUpgradable, CoreStatusEnum, ConsensusI {
     /* Internal functions */
 
     /**
-     * @notice Check if the core address is active.
+     * @notice Check if the core lifetime state is genesis or activated.
      * @param _core Core contract address.
      * Returns true if the specified address is a core.
      */
-    function isCore(address _core)
+    function isCoreInAction(address _core)
```

Comment:
`isCoreRunning` might capture `genesis | active`

Reply:
isCoreInAction renamed to isCoreActive which check `genesis` and `active`.

Reply:
I like @benjaminbollen 's proposal to have `isCoreRunning`. It's confusing to me to have `isCoreActive` function which checks that core is `active` or `genesis`. I would like to propose also to rename modifier to `onlyRunningCore`.